### DISCUSSION TITLE
Non-Git-Based Advanced Install

### DISCUSSION THREAD

**Discussion by Dulux-Oz** [acmesh-official/acme.sh#4671] [Q&A] [answered]
Hi All,

Is there a way &mdash; ***without using git*** &mdash; to do an advanced install?

I want to install acme.sh to a location that is **NOT** the default `/root/.acme`, and without having to install git (ie using curl and/or wget).

Cheers

Dulux-Oz

**Comment by neilpang** [comment, marked as answer]
yes,  you can download the source code, and decompress it.

https://github.com/acmesh-official/acme.sh/archive/refs/tags/3.0.6.zip

or

https://github.com/acmesh-official/acme.sh/archive/refs/tags/3.0.6.tar.gz
